### ISSUE TITLE
January 2023 Endgame

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#172091]
- *1/23* Code freeze for the endgame
- *1/27* [Endgame done](https://github.com/Microsoft/vscode/wiki/Development-Process#inside-an-iteration)
- *2/2* Expected release date (this may change)

##### Monday
- [x] Check that all queries in this issue use the current milestone **@roblourens**
- [x] Run [OSS tool](https://github.com/microsoft/vscode-build-tools/tree/main/distro-tools) **@roblourens**
- [x] Update links in the [Endgame notebooks](https://github.com/microsoft/vscode/blob/main/.vscode/notebooks/endgame.github-issues) to point to new milestone **@roblourens**
- [x] Code freeze at 5pm PT
- [x] Ensure we have a green build on all platforms at 5pm PT
- [x] [🔖All closed feature-requests](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+milestone%3A%22January+2023%22+label%3Afeature-request+-label%3Averification-needed+-label%3Aon-testplan+-label%3Averified+-label%3A*duplicate+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels) either have a `verification-needed` or `on-testplan` label
- [x] Create test plan items following the template [here](https://github.com/microsoft/vscode/wiki/Writing-Test-Plan-Items) by 6pm PT
- [x] Update your availability for testing here - https://tools.code.visualstudio.com/team-manifest **team**
  - [x] Update availability of testers in vacation. Double check N/A testers. **@roblourens**

##### Tuesday
- [x] Test plan items assigned (using https://tools.code.visualstudio.com/test-plan-items)
  - Run the tool multiple times to balance load if test items come in later and assignments are already made
  - [Assigned to you](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+assignee%3A%40me++label%3Atestplan-item+)
- [x] Test build starts at 7am CET
- [x] Test plan ready by 8am CET
- [x] [🔖Testing](https://github.com/issues?q=is%3Aopen+is%3Aissue+label%3Atestplan-item+repo%3Amicrosoft%2Fvscode+repo%3Amicrosoft%2Fvscode-internalbacklog+repo%3Amicrosoft%2Fvscode-dev+repo%3Amicrosoft%2Fvscode-remote-repositories-github+repo%3Amicrosoft%2Fvscode-remote-release+repo%3Amicrosoft%2Fvscode-js-debug+repo%3Amicrosoft%2Fvscode-pull-request-github+repo%3Amicrosoft%2Fvscode-livepreview+repo%3Amicrosoft%2Fvscode-python+repo%3Amicrosoft%2Fvscode-jupyter+repo%3Amicrosoft%2Fvscode-jupyter-internal+repo%3Adevcontainers%2Fcli+repo%3Adevcontainers%2Fci+repo%3Amicrosoft%2Fvscode-hexeditor)
- [x] [🔖Verification needed](https://github.com/issues?q=is%3Aissue+milestone%3A%22January+2023%22+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels+-label%3Aiteration-plan+-label%3Aendgame-plan+-label%3Atestplan-item+label%3Averification-needed+-label%3Averified+is%3Aclosed+-label%3Aon-testplan)

##### Wednesday
- [x] [🔖Testing](https://github.com/issues?q=is%3Aopen+is%3Aissue+label%3Atestplan-item+repo%3Amicrosoft%2Fvscode+repo%3Amicrosoft%2Fvscode-internalbacklog+repo%3Amicrosoft%2Fvscode-dev+repo%3Amicrosoft%2Fvscode-remote-repositories-github+repo%3Amicrosoft%2Fvscode-remote-release+repo%3Amicrosoft%2Fvscode-js-debug+repo%3Amicrosoft%2Fvscode-pull-request-github+repo%3Amicrosoft%2Fvscode-livepreview+repo%3Amicrosoft%2Fvscode-python+repo%3Amicrosoft%2Fvscode-jupyter+repo%3Amicrosoft%2Fvscode-jupyter-internal+repo%3Adevcontainers%2Fcli+repo%3Adevcontainers%2Fci+repo%3Amicrosoft%2Fvscode-hexeditor)
- [x] Remind team members to assign issues that they intend to fix to the current milestone
- [x] Fixing (self-assigned, milestone assigned)
- [x] [🔖Verification needed](https://github.com/issues?q=is%3Aissue+milestone%3A%22January+2023%22+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels+-label%3Aiteration-plan+-label%3Aendgame-plan+-label%3Atestplan-item+label%3Averification-needed+-label%3Averified+is%3Aclosed+-label%3Aon-testplan)

##### Thursday
- [x] Fixing (self-assigned, milestone assigned, no need for PR or review)
  - Increased scrutiny sets in due to testing being completed. Fixes pose a much higher risk
  - Move issues to the next month that can be deferred
- [x] [🔖Verification needed](https://github.com/issues?q=is%3Aissue+milestone%3A%22January+2023%22+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels+-label%3Aiteration-plan+-label%3Aendgame-plan+-label%3Atestplan-item+label%3Averification-needed+-label%3Averified+is%3Aclosed+-label%3Aon-testplan)
- [x] [🔖Verification](https://github.com/issues?q=is%3Aissue+milestone%3A%22January+2023%22+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels+-label%3Aiteration-plan+-label%3Aendgame-plan+-label%3Atestplan-item+label%3Abug+-label%3Aduplicate+-label%3A*duplicate+-label%3Ainvalid+-label%3Aas-designed+-label%3Aerror-telemetry+is%3Aclosed+-label%3Averified+-label%3Atestplan-item+-label%3Aon-testplan)

##### Friday
- [x] Build a `stable` build to ensure stable build is green **@roblourens**
- [x] Pause scheduled `insider` builds (includes vscode.dev publishing) **@roblourens**
- Satellite modules/npm packages ready, version updated, smoke tested
  - [x] yo generator **@aeschli**
  - [x] js-debug **@connor4312**
  - [x] Debug Adapter Protocol **@connor4312**
  - [x] vscode-debugadapter-node **@roblourens**
  - [x] Mock Debug **@roblourens**
  - [x] vscode-test-web **@aeschli**
- [x] Only candidate issues are open and assigned to [🔖milestone](https://github.com/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22January+2023%22+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels)
- [ ] All issues [🔖verified](https://github.com/issues?q=is%3Aissue+milestone%3A%22January+2023%22+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels+-label%3Aiteration-plan+-label%3Aendgame-plan+-label%3Atestplan-item+label%3Abug+-label%3Aduplicate+-label%3A*duplicate+-label%3Ainvalid+-label%3Aas-designed+-label%3Aerror-telemetry+is%3Aclosed+-label%3Averified+-label%3Aon-testplan)
- [x] All open PRs on the milestone [🔖merged or deferred](https://github.com/issues?q=is%3Apr+milestone%3A%22January+2023%22+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels+is%3Aopen)
- [x] Branch code to `release/<x.y>` after all expected fixes are in (latest 5PM PST) **@roblourens**
- [x] Branch distro to `release/<x.y>` after all expected fixes are in (latest 5PM PST) **@roblourens**
- [x] Branch vscode.dev to `release/<x.y>` after all expected fixes are in (latest 5PM PST) **@roblourens**
- [x] Bump up the version in `package.json` on `main` - **@roblourens**
- [x] Localization: Run [Update VS Code Branch](https://monacotools.visualstudio.com/Monaco/_build?definitionId=369) build with `release/*` as the VS Code Branch parameter (it's the default so you shouldn't have to change anything) **@roblourens**
- [x] [Create next milestone](https://github.com/microsoft/vscode-internalbacklog/milestones/new) and ensure that it has a due date. The created milestone and its due date will be automatically synced across our repos **@roblourens**
- [x] Announce `main` is open for business **@roblourens**
- [x] Fixing (PR + review required once branched - major bugs only - to be discussed in stand-up meeting, labeled as `candidate`)
- [ ] All release notes updated
  - release notes are collected in a file named *`v<Major>_<Minor>.md`* in this [repo directory](https://github.com/Microsoft/vscode-docs/blob/vnext/release-notes/)
  - [x] @aeschli
  - [ ] @alexdima
  - [x] @alexr00
  - [ ] @andreamah
  - [ ] @bhavyaus
  - [x] @bpasero
  - [x] @chrmarti
  - [x] @connor4312
  - [x] @dbaeumer
  - [ ] @deepak1556
  - [x] @egamma
  - [ ] @hediet
  - [x] @isidorn
  - [x] @joaomoreno
  - [x] @joyceerhl
  - [x] @jrieken
  - [x] @kieferrm
  - [x] @lramos15
  - [x] @lszomoru
  - [ ] @meganrogge
  - [x] @mjbvz
  - [x] @rebornix
  - [x] @roblourens
  - [x] @rzhao271
  - [ ] @sandy081
  - [ ] @sbatten
  - [ ] @tanhakabir
  - [x] @tylerleonhardt
  - [ ] @tyriar
- [x] Acknowledge pull requests in release notes. We acknowledge PRs from outside the team. We have improved [the tooling](https://tools.code.visualstudio.com/acknowledgement) so that the @roblourens can generate the pull request acknowledgment for all repositories at once. **@roblourens**
  - [x] `debug-adapter-protocol`, `inno-updater`, `jsonc-parser`, `language-server-protocol`, `lsif-node`, `vscode`, `vscode-codicons`, `vscode-css-languageservice`, `vscode-debugadapter-node`, `vscode-dev-containers`, `vscode-docs`, `vscode-emmet-helper`, `vscode-eslint`, `vscode-extension-samples`, `vscode-generator-code`, `vscode-hexeditor`, `vscode-html-languageservice`, `vscode-js-debug`, `vscode-js-debug-companion`, `vscode-js-profile-visualizer`, `vscode-jshint`, `vscode-json-languageservice`, `vscode-languageserver-node`, `vscode-livepreview`, `vscode-loader`, `vscode-lsif-extension`, `vscode-node-debug`, `vscode-node-debug2`, `vscode-pull-request-github`, `vscode-recipes`, `vscode-references-view`, `vscode-textmate`, `vscode-vsce`
- [x] Acknowledge [issue trackers](https://github.com/microsoft/vscode-internalbacklog/wiki/Community-Triage---Credits) from the community **@chrmarti**
- [x] Add notable fixes to the release notes **all**
- When done fixing/verifying and there are changes since last build at the end of day PT
  - [x] Build and manually release Insider from release/<x.y> for Code **@roblourens**
  - [x] Build and manually release Insider from release/<x.y> for vscode.dev (if required) **@roblourens**

##### Friday/Monday
- [x] Polish release notes **redmond**
- [x] Fixing (only critical bugs - no string changes)

##### Monday - Wednesday
- [x] [🔖milestone issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22January+2023%22+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels)
- [x] [🔖candidate issues](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3Acandidate+repo:microsoft/vscode%20repo:microsoft/vscode-remote-release%20repo:microsoft/vscode-js-debug%20repo:microsoft/vscode-pull-request-github%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-internalbacklog%20repo:microsoft/vscode-dev%20repo:microsoft/vscode-unpkg%20repo:microsoft/vscode-references-view%20repo:microsoft/vscode-anycode%20repo:microsoft/vscode-hexeditor%20repo:microsoft/vscode-extension-telemetry%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-remotehub%20repo:microsoft/vscode-settings-sync-server%20repo:microsoft/vscode-remote-repositories-github%20repo:microsoft/monaco-editor%20repo:microsoft/vscode-vsce%20repo:microsoft/vscode-dev-chrome-launcher%20repo:microsoft/vscode-emmet-helper%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-livepreview%20repo:microsoft/vscode-python%20repo:microsoft/vscode-jupyter%20repo:microsoft/vscode-jupyter-internal%20repo:microsoft/vscode-github-issue-notebooks%20repo:microsoft/vscode-l10n%20repo:microsoft/vscode-remote-tunnels)
- [x] Polish release notes **redmond**
- [x] Cherry-pick hand-picked and reviewed changes to `release/<x.y>` **@roblourens**
- [x] Build `Insider` from `release/<x.y>` **@roblourens**
- [x] Manually release `Insider` **@roblourens**
- [x] Build stable for all platforms as new candidate issues come in **@roblourens**
- [ ] Documentation updated
  - [x] @aeschli
  - [ ] @alexdima
  - [ ] @alexr00
  - [ ] @andreamah
  - [ ] @bhavyaus
  - [x] @bpasero
  - [x] @chrmarti
  - [x] @connor4312
  - [x] @dbaeumer
  - [ ] @deepak1556
  - [ ] @egamma
  - [ ] @hediet
  - [x] @isidorn
  - [x] @joaomoreno
  - [x] @joyceerhl
  - [ ] @jrieken
  - [x] @kieferrm
  - [x] @lramos15
  - [x] @lszomoru
  - [ ] @meganrogge
  - [x] @mjbvz
  - [x] @rebornix
  - [ ] @roblourens
  - [x] @rzhao271
  - [ ] @sandy081
  - [ ] @sbatten
  - [x] @tylerleonhardt
  - [ ] @tyriar
- [x] Run `scripts/test-documentation.sh|bat` and add file or fix issues if there are new colors that are not documented. **@roblourens**

> **Note:** The `Insiders` build needs to be in the wild for 24 hours before we can enter the last phase of the endgame. **@roblourens**

##### Wednesday/Thursday - Expected release day (this may change)
- [x] Build stable for all platforms **@roblourens**
- [x] Sanity check of installable bits ([server instructions](https://github.com/microsoft/vscode-remote-release/wiki/Sanity-Check-VS-Code-Servers))
  - [x] Windows 32 bit @rzhao271
    - [x] signed installer 32-bit
    - [x] signed user installer 32-bit 
    - [x] zip 32-bit  
    - [x] server 32-bit
    - [x] cli 32-bit
  - [x] Windows 64 bit **owner**
    - [x] signed installer 64-bit @joyceerhl
    - [x] signed user installer 64-bit @joyceerhl
    - [x] zip 64-bit @joyceerhl
    - [x] server 64-bit @rzhao271
    - [x] cli 64-bit @joyceerhl
  - [x] Windows ARM64 **@sandy081**
    - [x] signed installer ARM64
    - [x] signed user installer ARM64
    - [x] zip ARM64
    - [x] cli ARM64
  - [x] macOS
    - [x] Intel + Universal @lramos15 
    - [x] cli Intel @roblourens 
    - [x] Apple Silicon + Universal @kieferrm 
    - [x] cli Apple Silicon @kieferrm 
  - [x] Linux x64
    - [x] deb @connor4312
    - [x] rpm @rzhao271
    - [x] archives @connor4312
    - [x] snap (`sudo snap install --classic --dangerous <file>.snap`) @connor4312
    - [x] cli @connor4312
  - [x] Linux server **owner**
    - [x] x64 @roblourens
    - [x] x64 Alpine @rzhao271
    - [x] ARM32 @roblourens
    - [x] ARM64 @roblourens
    - [x] ARM64 Alpine @rzhao271
- [x] Publish website **@gregvanl**
- [x] Publish to stable **@roblourens**
- [x] Trigger the [vscode.dev Deploy pipeline](https://dev.azure.com/monacotools/Monaco/_build?definitionId=276) for the `prod` deployment target. Request approval from another team member at the necessary step. **@roblourens**
- [x] Create an official release **@roblourens**
  - [x] Create a tag (make sure you pull the release branch first): `git tag <x.y.z>`
  - [x] Push the tag: `git push origin <x.y.z>`
  - [x] Create a GitHub release: [Open the GitHub tags](https://github.com/microsoft/vscode/tags), and click far right `... > Create Release`. Use the correct title and description from our release notes. Also change the relative links for the key highlight list items to absolute links [Example](https://github.com/microsoft/vscode/releases/tag/1.39.0)
- [x] Twitter announcement **@chrisdias**
- [x] Enable scheduled `insider` builds **@roblourens**
- [x] [Publish @types/vscode](https://github.com/microsoft/vscode/wiki/Publish-vscode-types) **@roblourens**
- [ ] Close the milestone on [GitHub](https://github.com/microsoft/vscode/milestones) **@roblourens**
- [ ] If no Patch Tuesday release is planned, [let the Updates team know of that fact](https://github.com/microsoft/vscode-internalbacklog/wiki/Patch-Tuesday-E-Mail-Template) ([CSEClientPatching@microsoft.com](mailto:CSEClientPatching@microsoft.com)) **@roblourens**
- [ ] If a [recovery build](https://github.com/microsoft/vscode/wiki/Running-the-Endgame#recovery-build) is needed, proceed to https://github.com/microsoft/vscode/wiki/Endgame-Recovery-Template or https://github.com/microsoft/vscode-internalbacklog/wiki/Patch-Tuesday-Release-Endgame-Template/ **@roblourens**j

**Comment by lorand-horvath** [comment]
@roblourens Maybe I don't understand it, but I see you checked the Friday task `Bump up the version in package.json on main` as completed. However, the version in the file is still at 1.75.0 https://github.com/microsoft/vscode/blob/main/package.json
Shouldn't it say 1.76.0 already?

**Comment by lorand-horvath** [comment]
@roblourens Maybe I don't understand it, but I see you checked the Friday task `Bump up the version in package.json on main` as completed. However, the version in the file is still at 1.75.0 https://github.com/microsoft/vscode/blob/main/package.json
Shouldn't it say 1.76.0 already?

I've mentioned this problem before, in the October 2022 Endgame https://github.com/microsoft/vscode/issues/164400#issuecomment-1297202242 and at the time it was taken care of by @rebornix and @joaomoreno

**Comment by roblourens** [comment]
Thanks @lorand-horvath, I must have missed it while multitasking
